backend/server.js

```diff
@@ -35,13 +37,15 @@ app.use("/api/geo", geo);
 //  Serve static asset assets if in production
 if (process.env.NODE_ENV === "production") {
   // Set Static Folder
-  app.use(express.static("client/build"));
+  app.use(express.static("../client/build"));
 
   app.get("*", (req, res) => {
-    res.sendFile(path.resolve(__dirname, "client", "build", "index.html"));
+    res.sendFile(path.resolve(__dirname, "../client", "build", "index.html"));
```

Comment:
I would suggest to separate the client and backend containers. The client container can run the build files with nginx in front of it. I can help on this if you want

Reply:
yes this was also the intention of using docker compose, better with Nginx or some other static server (Caddy maybe?)